src/components/DashboardMenu/index.tsx

```diff
@@ -0,0 +1,22 @@
+import {FunctionComponent} from "react"
+import {Link} from "react-router-dom"
+import Paper from "../Paper"
+import "./styles.scss"
+
+const DashboardMenu: FunctionComponent<{
```

Comment:
Accordingly, to the new design - this component also will be reused on DAO dashboard. I will expand its props based on need when will work on that page